platforms/core-runtime/native/src/test/groovy/org/gradle/internal/nativeintegration/filesystem/services/NativePlatformBackedFileMetadataAccessorTest.groovy

```diff
@@ -32,7 +32,7 @@ import static org.gradle.test.fixtures.FileMetadataTestFixture.maybeRoundLastMod
 class NativePlatformBackedFileMetadataAccessorTest extends AbstractFileMetadataAccessorTest {
     @Override
     FileMetadataAccessor getAccessor() {
-        return new NativePlatformBackedFileMetadataAccessor(NativeServices.instance.get(Files.class))
+        return new NativePlatformBackedFileMetadataAccessor(NativeServices.getNative().get(Files.class))
```

Comment:
💭 This is actually `Native.get(Files.class)` on master, but changing it to `NativeServices.instance.get(Files.class)` broke the test, since there is no `Files` instance in `ServiceRegistry` for Windows